### PR TITLE
Added responsive image slider

### PR DESCRIPTION
**Pull request by sakshikhachane**
## Issue that this pull request solves

 Closes: #255 

## Proposed changes
Image slider 

### Brief description of what is fixed or changed
Added image slider which is responsive

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [`x`] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [ x`] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [`x`] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots
Canvasboard - Google Chrome 30-12-2020 14_38_31
Canvasboard - Google Chrome 30-12-2020 14_38_27
Canvasboard - Google Chrome 30-12-2020 14_38_09

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -133,6 +133,10 @@
...
         }
       }
-    }},
-  "defaultProject": "canvasboard"
-}
+    }
+  },
+  "defaultProject": "canvasboard",
+  "cli": {
+    "analytics": "f07f4b3b-52e3-4a35-8e03-3c8547e12d26"
```
Comment (goliakshay357) [inline angular.json:140]: Why this parameter?

File: src/app/components/home/home.component.html [hunk 2]
```diff
@@ -1,146 +1,192 @@
...
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Canvasboard</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
```
Comment (goliakshay357) [inline src/app/components/home/home.component.html:10]: All these packages are already there, why there is a need to add again?

File: angular.json (hunk 1 above)
Comment (sakshikhachane) [reply angular.json:140]: I haven't added it manually. It came when I downloaded the angular cli.

**Comment by sakshikhachane** [comment]
@goliakshay357 Modified the changes suggested.

File: angular.json [hunk 3]
```diff
@@ -133,6 +133,7 @@
           }
         }
       }
-    }},
+    }
+  },
   "defaultProject": "canvasboard"
```
Comment (goliakshay357) [inline angular.json:138]: angular.json shouldn't be pushed. Can you please ignore this file?
Thanks

File: src/app/components/home/home.component.html [hunk 4]
```diff
@@ -1,146 +1,188 @@
-<section id="header">
-  <div class="container">
-    <div class="row justify-content-md-center mt-4 text-center">
-      <div class="cb-image">
-        <img src="../../../assets/home/logo_bw.png" width="100" alt="Logo">
+<!DOCTYPE html>
```
Comment (goliakshay357) [inline src/app/components/home/home.component.html:1]: You cannot add html, head and body tags in a component. 
Please make sure you follow the syntax and principles
Thanks

**Comment by sakshikhachane** [comment]
@goliakshay357 I am sorry but I am not getting how to remove the angular.json. I tried to delete and then the checks were not successful. How should I proceed? Please guide.

**Comment by goliakshay357** [comment]
> @goliakshay357 I am sorry but I am not getting how to remove the angular.json. I tried to delete and then the checks were not successful. How should I proceed? Please guide.

Hey @sakshikhachane,
Please feel free to join our slack channel. You find amazing people out there and we are very happy to help you out. 
Slack invite: https://join.slack.com/t/canvasbird-workspace/shared_invite/zt-jjdml591-~SsL4vdSX9Mc_GUVIvYV7g
Thanks

**Comment by sakshikhachane** [comment]
> > @goliakshay357 I am sorry but I am not getting how to remove the angular.json. I tried to delete and then the checks were not successful. How should I proceed? Please guide.
> 
> Hey @sakshikhachane,
> Please feel free to join our slack channel. You find amazing people out there and we are very happy to help you out.
> Slack invite: https://join.slack.com/t/canvasbird-workspace/shared_invite/zt-jjdml591-~SsL4vdSX9Mc_GUVIvYV7g
> Thanks

The link is no longer active :(

**Comment by goliakshay357** [comment]
> > > @goliakshay357 I am sorry but I am not getting how to remove the angular.json. I tried to delete and then the checks were not successful. How should I proceed? Please guide.
> > 
> > 
> > Hey @sakshikhachane,
> > Please feel free to join our slack channel. You find amazing people out there and we are very happy to help you out.
> > Slack invite: https://join.slack.com/t/canvasbird-workspace/shared_invite/zt-jjdml591-~SsL4vdSX9Mc_GUVIvYV7g
> > Thanks
> 
> The link is no longer active :(
Here's the new invite link:
https://join.slack.com/t/canvasbird-workspace/shared_invite/zt-kdh1jv5x-_UH2fPZE5FIt4ZaHy2M2TQ

Thanks

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@sakshikhachane please don't delete these files `.gitignore` and `angular.json` Instead you just had to revert your changes by using `git checkout HEAD -- <file-name>` 
Reference link -https://stackoverflow.com/questions/7147270/hard-reset-of-a-single-file

Also instead of putting the left and right arrow key on the slider itself, let's keep it below the slider, aligned horizontally centered, it would look cleaner that way.

**Pull request closed without merging**